pkg/datapath/loader/base.go

```diff
@@ -187,12 +187,6 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 		args[initArgIPSec] = "false"
 	}
 
-	if !option.Config.InstallIptRules && option.Config.Masquerade {
-		args[initArgMasquerade] = "true"
```

Comment:
Maybe we should rename `initArgMasquerade` to `initArgMasqueradeUnused` or `_` or reshuffle the set of init.sh args entirely? Will the subsequent PR restore use of this argument?

Reply:
I've just got rid of it.